modules/spring-data-2.0-ext/src/main/java/org/apache/ignite/springdata20/repository/support/IgniteRepositoryFactoryBean.java

```diff
@@ -22,6 +22,7 @@
 import org.apache.ignite.configuration.IgniteConfiguration;
 import org.apache.ignite.springdata20.repository.IgniteRepository;
 import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
Unused import. Intellij IDEA supports feature to find unused imports, please check it.

Reply:
Done.